webpack/index.js

```diff
@@ -1,13 +1,24 @@
 'use strict';
 
-const
-    env = require('../env'),
-    layout = require('../layout'),
-    logEnv = require('../env/log'),
-    loaders = require('./loaders'),
-    autoprefixer = require('autoprefixer');
+var env = require('../env');
```

Comment:
vars usage violates our style-guide,
Please use const where possible, and let when mutation is neseccary